ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -34,30 +34,51 @@
 | -------- | -------- | -------- | -------- |
 {# ethernet interfaces #}
 {# ------- #}
-{%      if ethernet_interfaces is defined and ethernet_interfaces is not none %}
+{%      if ethernet_interfaces is arista.avd.defined %}
```

Comment:
We don't require this test ```arista.avd.defined``` as we are using ```arista.avd.natural_sort``` filter which by default uses ```arista.avd.defined``` test. 

Let's use ```is defined``` test before ```natural_sort``` filter only if you want to apply some logic or code in between those two steps.

Reply:
Fixed.